services/provider/server/server.go

```diff
@@ -731,6 +731,8 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 	replicationEnabled := len(scp.Items) > 0
 	replicationID := req.StorageClaimName
 
+	clientProfile := util.CalculateMD5Hash(req.StorageClaimName)
```

Comment:
this could become problematic when we refactor the responsibility of provider & client side code, better be consistent and let client-op set this.

Reply:
It was already decided that we are going to set clusterID from the provider